src/test/resources/recipes/MultipleDereferencesRecipes.java

```diff
@@ -8,8 +8,30 @@
 import org.openrewrite.java.template.Primitive;
 import org.openrewrite.java.tree.*;
 
-public final class MultipleDereferencesRecipes {
-    public static final class StringIsEmptyRecipe extends Recipe {
+import java.util.Arrays;
+import java.util.List;
+
+public final class MultipleDereferencesRecipes extends Recipe {
+
+    @Override
+    public String getDisplayName() {
+        return "Refaster recipes for `foo.MultipleDereferences`";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Refaster template recipes for `foo.MultipleDereferences`.";
```

Comment:
Should we possibly also here include the source code as Markdown?

Reply:
I'm afraid that would result in a very long blob of text for some of the recipes I anticipate here. Take for instance [the Plexus `StringUtils` methods](https://github.com/codehaus-plexus/plexus-utils/blob/master/src/main/java/org/codehaus/plexus/util/StringUtils.java) we hope to replace with recipes; that's a _long_ list that I think would best be documented on each of the individual recipes.

I do understand there's some tension with whether that recipe would have visible individual recipes, but since we also plan to add a `@DocumentExample` to the docs, then I think including the recipe source here could be too much.

Reply:
OK. Let's keep it like this for now. We may want to add an annotation with which we can control the generation of the display name and description.